README.md

```diff
@@ -5,19 +5,19 @@ This repository contains various code samples for ETL tasks written in Ballerina
 Each subdirectory contains relevant code and instructions for the corresponding task. Explore the subdirectories for detailed code samples and documentation.
 
 #### Data Extraction (E)
-- [Extract Data from Database](extract-data-from-database): Sample code for extracting customer orders from database.
-- [Extract Information From Unstructured Data](extract-information-from-unstructured-data): Sample code for extracting required information from unstructured data using AI.
-- [Extract Data from GitHub](extract-data-from-github): Sample code samples for extracting data from GitHub.
+- [Extract data from database](extract-data-from-database): Sample code for extracting customer orders from database.
+- [Extract information from unstructured data](extract-information-from-unstructured-data): Sample code for extracting required information from unstructured data using AI.
+- [Extract data from gitHub](extract-data-from-github): Sample code samples for extracting data from GitHub.
 
 #### Data Transformation (T)
-- [Remove Duplicates](remove-duplicates): Sample code for deduplication.
-- [Remove Approximate Duplicates](remove-approximate-duplicates): Sample code for approximate deduplication.
-- [Error Correction](error-correction): Sample code for error correction in data.
-- [Clean and Standardize Data](clean-and-standardize-data): Sample code for cleaning and standardizing data.
-- [Data Mapper](data-mapper): Sample code for data mapping between different structures.
-- [Enrich Data](enrich-data): Sample code to enrich data.
+- [Remove duplicates](remove-duplicates): Sample code for deduplication.
+- [Remove approximate duplicates](remove-approximate-duplicates): Sample code for approximate deduplication.
+- [Refine data using ai](refine-data-using-ai): Sample code for error correction in data.
```

Comment:
AI should also be fine

Reply:
updated in [commit](https://github.com/ballerina-guides/etl-samples/pull/15/commits/efa507c9dc10531c031b38c0f5760f42265cad32).